### PR TITLE
Scorecard bump kuttl

### PR DESCRIPTION
**Pull request by jmccormick2001**
**Description of the change:**
bump the kuttl base image version to v0.5.2, also execute kuttl with the --suppress-log=events flag

**Motivation for the change:**
this PR is necessary to add
the --suppress-log=events kuttl CLI flag which we do in order to avoid running kuttl tests with 'list' on events RBAC permissions.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by jmccormick2001** [comment]
> Is there a way a user can configure the `scorecard-test-kuttl` image to not suppress event logging, say if they add the appropriate RBAC rules? If not, is there an easy way (either in scorecard config or in the kuttl config directly) to add this configuration option?

yep, I can go that way, in the kuttl-test.yaml, it appears to support adding suppressLog: events there, so that way users can add that if they want events logged for some reason.   I made that change and also updated the sample kuttl-test.yaml to have suppressLog=events as the default.

File: internal/scorecard/testdata/bundle/tests/scorecard/kuttl/kuttl-test.yaml [hunk 1]
```diff
@@ -3,3 +3,4 @@ kind: TestSuite
 parallel: 4
 timeout: 120
 startControlPlane: false
+suppressLog: events
```
Comment (estroz) [inline internal/scorecard/testdata/bundle/tests/scorecard/kuttl/kuttl-test.yaml:6]: :+1:

**Comment by estroz** [comment]
@joelanford something to change post-v1.0?

/hold

**Comment by estroz** [comment]
This is fine for v1.0

/hold cancel

**Comment by jmccormick2001** [comment]
I'm going to close this PR and submit a new PR that just has the version bump and not the extra supress-log work.

**Pull request closed without merging**